www/js/app.js

```diff
@@ -485,9 +553,9 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
      * If Service Worker is not available, the attributes of the memory cache are returned instead
      * @returns {Promise<Object>} A Promise for an object with cache attributes 'type', 'description', and 'count'
      */
-    function getCacheAttributes() {
+    function getAssetsCacheAttributes() {
         return new Promise(function (resolve, reject) {
-            if (contentInjectionMode === 'serviceworker') {
+            if (params.contentInjectionMode === 'serviceworker' && navigator.serviceWorker.controller) {
```

Comment:
It might be not strictly necessary here, but you might test `navigator.serviceWorker` too, before testing `navigator.serviceWorker.controller`